src/utils/characterSet.ts

```diff
@@ -0,0 +1,24 @@
+import emojiRegex                     from "emoji-regex";
+import { characterSet, CharacterSet } from "../base";
+
+
+export function findCharacterSet(name: string): CharacterSet {
+    const label = name.substring(0, name.lastIndexOf('.'));
+    // digit only
+    if (!isNaN(Number(label)))
```

Comment:
Should probably instead check against a regex of `[0-9]+`, so we don't allow hyphens.